RawBencher.Core/RawBencher.Core.csproj

```diff
@@ -36,10 +36,6 @@
       <Compile Include="..\RawBencher\SalesOrderHeader.cs" Link="SalesOrderHeader.cs" />
     </ItemGroup>
 
-    <ItemGroup>
```

Comment:
why is this removed? Isn't it required to make sure the files compile?

Reply:
Hi Frans, I am having conflict on my local machine, so I did a copy+paste of file in the solution (which is wrong). I need to redo this. I have fixed it on my local machine by manually updating the 'csproj' file.